### ISSUE TITLE
Return keyboard focus to error in facility form

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4094]
## Area or Component
Facility form

## Quick Summary
On an attempted submission with errors, the keyboard focus should return to the first form error. Instead it stays on the bottom of the form.

## Accessibility Violation
https://dequeuniversity.com/class/forms/form-validation/focus-first-error

## Action Requested
Return keyboard focus to the first error on facility submission. See #4044 for more information.

**Comment by BobanL** [comment]
It would be valuable to make a util function as [mentioned by Emma](https://github.com/CDCgov/prime-simplereport/pull/4205#pullrequestreview-1086170904)
